src/home/ActionsCarousel.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text } from 'react-native'
+import Card from 'src/components/Card'
+import Touchable from 'src/components/Touchable'
+import { FiatExchangeFlow } from 'src/fiatExchanges/utils'
+import { HomeActionName } from 'src/home/types'
+import HomeActionsAdd from 'src/icons/home-actions/Add'
+import HomeActionsReceive from 'src/icons/home-actions/Receive'
+import HomeActionsRequest from 'src/icons/home-actions/Request'
+import HomeActionsSend from 'src/icons/home-actions/Send'
+import HomeActionsSwap from 'src/icons/home-actions/Swap'
+import HomeActionsWithdraw from 'src/icons/home-actions/Withdraw'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+
+function ActionsCarousel() {
+  const { t } = useTranslation()
+
+  const actions = [
+    {
+      name: HomeActionName.Send,
+      title: t('send'),
+      icon: <HomeActionsSend />,
+      onPress: () => {
+        navigate(Screens.Send)
+      },
+    },
+    {
+      name: HomeActionName.Receive,
+      title: t('homeActions.receive'),
+      icon: <HomeActionsReceive />,
+      onPress: () => {
+        navigate(Screens.QRNavigator)
+      },
+    },
+    {
+      name: HomeActionName.Add,
+      title: t('homeActions.add'),
+      icon: <HomeActionsAdd />,
+      onPress: () => {
+        navigate(Screens.FiatExchangeCurrency, {
+          flow: FiatExchangeFlow.CashIn,
+        })
+      },
+    },
+    {
+      name: HomeActionName.Swap,
+      title: t('homeActions.swap'),
+      icon: <HomeActionsSwap />,
+      onPress: () => {
+        navigate(Screens.SwapScreen)
+      },
+    },
+    {
+      name: HomeActionName.Request,
+      title: t('request'),
+      icon: <HomeActionsRequest />,
+      onPress: () => {
+        navigate(Screens.Send, { isOutgoingPaymentRequest: true })
+      },
+    },
+    {
+      name: HomeActionName.Withdraw,
+      title: t('withdraw'),
+      icon: <HomeActionsWithdraw />,
+      onPress: () => {
+        navigate(Screens.FiatExchange)
+      },
+    },
+  ]
+
+  return (
+    <ScrollView
+      horizontal={true}
+      showsHorizontalScrollIndicator={false}
+      contentContainerStyle={styles.carouselContainer}
+      testID={'HomeActionsCarousel'}
+    >
+      {actions.map(({ name, title, icon, onPress }) => (
+        <Card style={styles.card} shadow={null} key={`HomeAction-${name}`}>
```

Comment:
We had a discussion with Nitya in refinement specifically on this component having the new 'Bar Shadow'. I see that there is no shadow in the figma right now, but can you double check that thats intentional?

Reply:
yes, no shadow for these, but there are some shadow changes for other components. Will do those in a separate PR